src/Services/SecurityServices.php

```diff
@@ -11,14 +11,13 @@ public function searchUser(string $email, string $password)
     {
```

Comment:
**SecurityServices** -> change name -> **AuthenticationService**| **AuthService**

**SecurityServices**::**searchUser**() -> **AuthService**::**authenticate**()

Keep input parameters
Change result type to User
If authentication fails -> throw exception -> let it bubble up.
It should be the responsibility of this method to encode the password.